src/js/blue-button/components/DownloadLink.jsx

```diff
@@ -51,7 +51,7 @@ class DownloadLink extends React.Component {
 
 DownloadLink.propTypes = {
   name: React.PropTypes.string,
-  contentType: React.PropTypes.string,
+  docType: React.PropTypes.string,
```

Comment:
`docType` seems like it should be a required prop so that the `requestUrl` doesn't get something strange. And probably limit it to specific doc types with `React.PropTypes.oneOf`.